core/src/main/scala/chisel3/internal/firrtl/Converter.scala

```diff
@@ -19,6 +19,7 @@ private[chisel3] object Converter {
       val (fmts, args) = pables.map(p => unpack(p, ctx)).unzip
       (fmts.mkString, args.flatten.toSeq)
     case PString(str) => (str.replaceAll("%", "%%"), List.empty)
+    case PStringNew(str) => (str, List.empty)
```

Comment:
This is needed because for cf API we do not need to do this - since the user / caller has to explicitly handle this. The previous "p" API  it was not done by user.

Reply:
This is no longer needed as we handle literal percent by emitting `Percent` type in PercentSlitter function.